spec/instance_solr_doc_spec.rb

```diff
@@ -73,6 +73,14 @@
       allow(isd).to receive(:sparql).and_return(sparql_conn)
       expect(doc_hash[:title_245a_search]).to eq 'foo'
     end
+    it 'includes language field' do
+      allow(isd).to receive(:instance_uri_to_ckey).and_return('666')
+      solutions = RDF::Query::Solutions.new
+      solutions << RDF::Query::Solution.new(lang: RDF::URI('http://id.loc.gov/vocabulary/languages/ita'))
+      sparql_conn = double('sparql client', query: solutions)
+      allow(isd).to receive(:sparql).and_return(sparql_conn)
+      expect(doc_hash[:language]).to eq ['Italian']
+    end
```

Comment:
See #32 where the specs at this level of the code can simply check that this 'parent' code is calling a method from the 'child' module, i.e. mock the call to `language_values` and check that it is called once.  All the details on how it behaves are all specs on the `LanguageField` module.